### PR TITLE
Case 21408:  Quest demo - Fixing the gamma correction

### PR DESCRIPTION
**Pull request by samcake**
https://highfidelity.manuscript.com/f/cases/21408/Quest-demo-Fixing-the-gamma-correction

This pr address the gamma correction of the scene  rendered frame when blit in the composite buffer.

We removed the the display-plugin slftolinear shader and introduced 2 shaders in the gpu library 
one to to do the linearToSrgb conversion and the sRGBToLinear conversion.

Separated the hud pipeline in the OpenGLDIsplayPLugin from the simplePipeline to address the various case encountered.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17561/HighFidelity-Beta-PR15010-df6d8f3.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17561/HighFidelity-Beta-PR15010-df6d8f3.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17561/HighFidelity-Beta-Interface-PR15010-df6d8f3.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17561/HighFidelity-Beta-PR15010-df6d8f3.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17561/HighFidelity-Beta-Interface-PR15010-df6d8f3.exe

**Comment by samcake** [comment]
build this please

File: libraries/gpu/src/gpu/DrawTextureGammaLinearToSRGB.slf [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<@include gpu/Config.slh@>
+<$VERSION_HEADER$>
+//  Generated on <$_SCRIBE_DATE$>
+//
+//  DrawTexture.frag
```
Comment (jherico) [inline libraries/gpu/src/gpu/DrawTextureGammaLinearToSRGB.slf]: fix comment date and name

File: libraries/gpu/src/gpu/DrawTextureGammaSRGBToLinear.slf [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<@include gpu/Config.slh@>
+<$VERSION_HEADER$>
+//  Generated on <$_SCRIBE_DATE$>
+//
+//  DrawTexture.frag
```
Comment (jherico) [inline libraries/gpu/src/gpu/DrawTextureGammaSRGBToLinear.slf]: fix comment date and name

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17563/HighFidelity-Beta-PR15010-df6d8f3.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17563/HighFidelity-Beta-Interface-PR15010-df6d8f3.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17566/HighFidelity-Beta-PR15010-b563f21.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17566/HighFidelity-Beta-Interface-PR15010-b563f21.dmg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17566/HighFidelity-Beta-PR15010-b563f21.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17566/HighFidelity-Beta-PR15010-b563f21.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17566/HighFidelity-Beta-Interface-PR15010-b563f21.exe

**Pull request merged**